esmvaltool/recipes/clouds/recipe_bock24acp_fig3-4_maps.yml

```diff
@@ -0,0 +1,218 @@
+  # CMIP5
+  - {dataset: bcc-csm1-1, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: bcc-csm1-1-m, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GFDL-ESM2G, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GFDL-ESM2M, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GISS-E2-H, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: GISS-E2-R, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: inmcm4, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: IPSL-CM5B-LR, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: MIROC5, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: MRI-CGCM3, ensemble: r1i1p1, project: CMIP5}
+  - {dataset: NorESM1-M, ensemble: r1i1p1, project: CMIP5}
+
+
+preprocessors:
+
+  lat_lon_mmm:
+    custom_order: true
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+    multi_model_statistics:
+      span: full
+      statistics: [mean]
+    climate_statistics:
+      operator: mean
+
+  lat_lon:
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+    climate_statistics:
+      operator: mean
+
+
+diagnostics:
+
+  # Figure 3abc
+  clt_lat_lon: &lat_lon_diag
+    description: comparison of geographical maps
+    variables:
+      ECS_high: &var_clt
+        short_name: clt
+        preprocessor: lat_lon_mmm
+        mip: Amon
+        project: CMIP6
+        exp: historical
+        ensemble: r1i1p1f1
+        grid: gr
+        <<: *years_hist
+        additional_datasets: *datasets_ecs_high
+      ECS_med:
+        <<: *var_clt
+        additional_datasets: *datasets_ecs_med
+      ECS_low:
+        <<: *var_clt
+        additional_datasets: *datasets_ecs_low
+    scripts:
+      lat_lon:
+        script: clouds/clouds_ecs_groups_maps.py
+        group_by: variable_group
+        plot_each_model: false
+        reference: false
+
+
+  # Figure 3def
+        #  lwp_lat_lon:
+        #    <<: *lat_lon_diag
+        #    variables:
+        #      ECS_high:
+        #        <<: *var_clt
+        #        short_name: lwp
+        #        derive: true
+        #        additional_datasets: *datasets_ecs_high
+        #      ECS_med:
+        #        <<: *var_clt
+        #        short_name: lwp
+        #        derive: true
+        #        additional_datasets: *datasets_ecs_med
+        #      ECS_low:
+        #        <<: *var_clt
+        #        short_name: lwp
+        #        derive: true
+        #        additional_datasets: *datasets_ecs_low
+
+
+  # Figure 3ghi
+        #  iwp_lat_lon:
+        #    <<: *lat_lon_diag
+        #    variables:
+        #      ECS_high:
+        #        <<: *var_clt
+        #        short_name: clivi
+        #        additional_datasets: *datasets_ecs_high
+        #      ECS_med:
+        #        <<: *var_clt
+        #        short_name: clivi
+        #        additional_datasets: *datasets_ecs_med
+        #      ECS_low:
+        #        <<: *var_clt
+        #        short_name: clivi
+        #        additional_datasets: *datasets_ecs_low
```

Comment:
Remove commented code?

Reply:
I want to keep them. Therefore, I wrote already a note at the beginning of the recipe. When the Iris issue is fixed, it could be used directly.